confd/templates/storm/storm.yaml.tmpl

```diff
@@ -33,4 +33,4 @@ supervisor.slots.ports:
 nimbus.childopts: "-Xmx128m"
 ui.childopts: "-Xmx128m"
 supervisor.childopts: "-Xmx128m"
-worker.childopts: "-Xmx256m"
+worker.childopts: "-Xmx280m"
```

Comment:
Why is it here?

Reply:
Heap size was increased because ClassGraph crashed with an OutOfMemory error.